client/src/pages/Landing/Landing.tsx

```diff
@@ -75,25 +70,33 @@ export default function Landing() {
             )}
           >
             <div className="flex flex-col">
-              <div className={classnames("font-bold font-headline tracking-[-0.075rem] sm:tracking-[-0.1rem]", "text-3xl sm:text-4xl md:text-5xl xl:text-6xl")}>
-                <div className="sm:whitespace-nowrap text-on-surface">A better way to </div>
+              <div className={classnames("font-semibold tracking-[-0.075rem] sm:tracking-[-0.1rem]", "flex flex-col sm:items-center lg:items-start")}>
+                <div
+                  className={classnames(
+                    "font-headline whitespace-nowrap text-primary bg-surface-variant text-xl md:text-2xl",
+                    "py-4 px-6 rounded-full w-fit -rotate-6 text-center",
+                  )}
+                >
+                  A better way to
+                </div>
                 <div
                   className={classnames(
-                    styles["title-highlight"],
-                    "text-primary-60 sm:whitespace-nowrap",
+                    "font-display uppercase mt-6",
+                    "text-on-background",
                     "text-5xl sm:text-6xl md:text-7xl xl:text-7xl",
-                    "pb-2"
+                    "pb-2",
+                    styles.shadow
                   )}
                 >
-                  organize deadlines
+                  organize<br />deadlines
                 </div>
               </div>
 
-              <div className={classnames("text-on-surface-variant mt-8", "mr-8 sm:mr-0 lg:mr-20", landingText)}>
-                Lifeline takes the stress out of managing important dates for your
-                courses. Start saving more time and always keep track of your
-                deadlines in just a few clicks.
-                Start for free today to make your scheduling a breeze.
+              <div className={classnames("text-on-surface-variant mt-4", "mr-8 sm:mr-0 lg:mr-20", landingText)}>
+                Lifeline takes the stress out of managing important dates and tasks for your courses.
+                In just a few clicks, compile your deadlines from your course outlines,
```

Comment:
I am not a native speaker, so I could totally be wrong on this. But I think it sounds better to say "In just a few clicks, it compiles your deadlines...", instead of just saying "compile....."?

Reply:
Text changes are from #122 so you might want to raise these comments below the issue.